/* Rules used: r0, r1, r4, r6, r7, r25, r32 r61 */
// r0
class T4 {
    public static void main(String[] arg) { // r1, r61
        int i; // r6
        i = i + 2; // r25, r32
    }
}
class T4 { // r4
    public int something(){ // r7
        int i; // r6
        i = 0; // r25
        return i;
    }
}